### PR TITLE
Fix panic on `/raftz` when shutting down

### PR DESCRIPTION
**Pull request by neilalexander**
Signed-off-by: Neil Twigg <neil@nats.io>

### CODE REVIEW

**Review by wallyqs — APPROVED** [review]
LGTM

File: server/monitor.go [hunk 1]
```diff
@@ -3788,7 +3788,13 @@ func (s *Server) HandleRaftz(w http.ResponseWriter, r *http.Request) {
 	gfilter := r.URL.Query().Get("group")
 	afilter := r.URL.Query().Get("acc")
 	if afilter == "" {
```
Comment (derekcollison) [inline server/monitor.go:3790]: _EMPTY_

**Review by derekcollison — APPROVED** [review]
LGTM - one nit

File: server/monitor.go (hunk 1 above)
Comment (neilalexander) [reply server/monitor.go]: fixed

**Pull request merged**